logger/logger.go

```diff
@@ -71,3 +44,84 @@ func GetLevel(levelStr string) (Level, error) {
 	}
 	return InfoLevel, fmt.Errorf("Unknown Level String: '%s', defaulting to NoLevel", levelStr)
 }
+
+// basic Logger is the default global
+var globalLogger Logger = NewLogger()
+
+func SetGlobalLogger(logger Logger) {
+	globalLogger = logger
+}
+
+func SetGlobalLevel(lvl Level) {
+	globalLogger.SetLevel(lvl)
+}
+
+func Info(args ...interface{}) {
+	globalLogger.Log(InfoLevel, "", args, nil)
+}
+func Infof(template string, args ...interface{}) {
+	globalLogger.Log(InfoLevel, template, args, nil)
+}
+func Infow(msg string, fields Fields) {
+	globalLogger.Log(InfoLevel, msg, nil, fields)
+}
+
+func Trace(args ...interface{}) {
+	globalLogger.Log(TraceLevel, "", args, nil)
+}
+func Tracef(template string, args ...interface{}) {
+	globalLogger.Log(TraceLevel, template, args, nil)
+}
+func Tracew(msg string, fields Fields) {
+	globalLogger.Log(TraceLevel, msg, nil, fields)
+}
+
+func Debug(args ...interface{}) {
+	globalLogger.Log(DebugLevel, "", args, nil)
+}
+func Debugf(template string, args ...interface{}) {
+	globalLogger.Log(DebugLevel, template, args, nil)
+}
+func Debugw(msg string, fields Fields) {
+	globalLogger.Log(DebugLevel, msg, nil, fields)
+}
+
+func Warn(args ...interface{}) {
+	globalLogger.Log(WarnLevel, "", args, nil)
+}
+func Warnf(template string, args ...interface{}) {
+	globalLogger.Log(WarnLevel, template, args, nil)
+}
+func Warnw(msg string, fields Fields) {
+	globalLogger.Log(WarnLevel, msg, nil, fields)
+}
+
+func Error(args ...interface{}) {
+	globalLogger.Log(ErrorLevel, "", args, nil)
+}
+func Errorf(template string, args ...interface{}) {
+	globalLogger.Log(ErrorLevel, template, args, nil)
+}
+func Errorw(msg string, err error) {
+	globalLogger.Error(ErrorLevel, msg, nil, err)
+}
+
+func Panic(args ...interface{}) {
```

Comment:
We don't need Panic methods if we have Fatal.

Reply:
we offer `PanicLevel` .  how dev can use it without this method?